config/app/packages/netgen_information_collection.yaml

```diff
@@ -18,4 +18,4 @@ netgen_information_collection:
                 default:
                     - database
                     - email
-                    # - ngsite.info_collection.action.newsletter
+                    - Netgen\Bundle\SiteBundle\InfoCollection\Action\NewsletterAction
```

Comment:
The service name should be kept as in example since the service is not registered via FQCN in site bundle.